configs/default/IFRC-IFF4_AIO.config

```diff
@@ -81,7 +80,6 @@ feature RX_SERIAL
 feature TELEMETRY
 feature LED_STRIP
 feature DISPLAY
-feature OSD
```

Comment:
Not sure if this is the right way to do this - this will mean that existing users of the IFF4_AIO board will get this installed on a firmware upgrade, and then their OSD will stop working, and they will open a support case.
Probably a better idea to leave this as it is, and add a new target `IFRC-IFF4_AIO_DJI` or similar.